lib/src/steps/include_test_step.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:automated_testing_framework/automated_testing_framework.dart';
+import 'package:meta/meta.dart';
+
+/// Test step that allows to execute all the steps from another [Test].
+class IncludeTestStep extends TestRunnerStep {
+  IncludeTestStep({
+    this.suiteName,
+    @required this.testName,
```

Comment:
Please add an optional version.  If not set, use the latest.  But if set, use the explicit version set.